webpack.config.js

```diff
@@ -14,9 +14,7 @@ module.exports = {
       {
         test: /\.(scss)$/,
         use: [
-          {
-            loader: "style-loader" // inject CSS to page
-          },
+          MiniCssExtractPlugin.loader, // extract CSS in single file
```

Comment:
You need to know the reason. I guess the `MiniCssExtractPlugin` should be much slower than `style-loader` since it has to extract CSS from different files and combine them together. You can only use this way for a prototype app.

Reply:
I find build time and browser load time similar on `MiniCssExtractPlugin` and `style-loader`.

`style-loader`

**build time 3029ms**
![image](https://user-images.githubusercontent.com/13488449/61920488-23fcb100-af9d-11e9-8203-477cc354743e.png)

`MiniCssExtractPlugin`

**build time 3910ms**
![image](https://user-images.githubusercontent.com/13488449/61920741-f49a7400-af9d-11e9-9221-cb9159b9be62.png)

---

`style-loader`
**browser load time 880ms**
![image](https://user-images.githubusercontent.com/13488449/61920645-ac7b5180-af9d-11e9-95e3-04629a86211c.png)

`MiniCssExtractPlugin`
**browser load time 824ms**
![image](https://user-images.githubusercontent.com/13488449/61920697-d59be200-af9d-11e9-84d6-b74e0672029a.png)

Reply:
Definitely, It makes sense to me too. As I mentioned, it's fine if it's a prototype app. While you have 100+ js files which also imports CSS files, the extraction process would be much slower than your current measurement.